src/main/target/STM32_UNIFIED/target.h

```diff
@@ -51,10 +51,16 @@
 #define TARGET_IO_PORTE 0xffff
 #define TARGET_IO_PORTF 0xffff
 
-#elif defined(STM32F411)
+#elif defined(STM32F411) || defined(STM32F411_CC2500)
+#if defined(STM32F411)
 #define TARGET_BOARD_IDENTIFIER "S411"
 
 #define USBD_PRODUCT_STRING     "Betaflight STM32F411"
+#else
+#define TARGET_BOARD_IDENTIFIER "SF4C"
```

Comment:
Yeah this the exactly case i was thinking about, how can a noob tell this f4 is not f40x but f411 with cc2500 RF chip?

Reply:
the user continues using same target as beefore. Processor name shoul by change in configs.

Reply:
All CC2500 targets includes RX in the name

Reply:
No, the user in this case does not use this target, or say there's no target for specific RF chip before.

Reply:
The user should use same target as before, MATEKF411RX for example, it will use SF4C.

Reply:
I mean, without any additional description for unified target's system change, how can a user switch to this target?

Reply:
Let's say I'm on matekf411rx which uses f411 unified target, how can I know that the old unified target doesn't work for me?

Reply:
No switch needed, we shoul update all RX targets to use this new STM32 cc2500 based.

Reply:
Rabbit whole, if that then the user who wanna down grade to 4.1 will find there's no way to doofly downgrade cause the config already changed the targeting unified target unless they do this manually.

Reply:
Just flash with full chip erase like before.

Reply:
I cant see any problem with this, Unified target system is a really advantage.

Reply:
If you change the targeting unified target in the unified target config files, to this new unified target like STM32F411_CC2500, then that target shown in the configurator will only have 4.3 as the only version, cause in 4.2 we didn't have this CC2500 variant unified target. History problem.

Reply:
Ok yes right, we can search for a simply solution for that in configurator side.

Reply:
keep old configurations and only apply if it is not 4.3

Reply:
Then no problem for me if configurator side can also make some changes.

Reply:
Target board identifiers don't mean much in the case of a unified target, probably should be renamed to firmware identifier now.